README.md

```diff
@@ -154,6 +199,42 @@ produce early errors because of recursive application of the first rule.
 - Let _targetName_ be ? Get(_target_, `"name"`).
 - If Type(_targetName_) is not `String`, then let _targetName_ be the empty string.
 - Let _status_ be ? SetFunctionName(_F_, _targetName_, `"bound"`).
+- Assert: _status_ is never used.
+
+
+----
+
+    BindExpression:
+        LeftHandSideExpression :: new
+
+- Let _targetReference_ be the result of evaluating _LeftHandSideExpression_.
+- Assert: Type(_targetReference_) is `Reference`.
```

Comment:
Done, but I'm guessing you're talking about the result of evaluating *CallExpression*s. Am I correct?